src/T2DMap.h

```diff
@@ -165,6 +165,7 @@ class T2DMap : public QWidget
     bool mSizeLabel;
     bool isCenterViewCall;
     QString mHelpMsg;
+    int mapInfoOverrideCallback;
```

Comment:
:beetle: This is what was causing the problems - it was not being initialised and anything other then a value of `0` is going to cause Mudlet to use a (likely non-existent) callback function with the index that this `int` gets *uninitialised* to!
```suggestion
    int mapInfoOverrideCallback = 0;
```